proto/aggkit/prover/v1/aggchain_proof_generation.proto

```diff
@@ -55,6 +65,18 @@ message GenerateAggchainProofResponse {
   bytes custom_chain_data = 5;
 }
 
+// The optimistic aggchain proof response message.
+message GenerateOptimisticAggchainProofResponse {
```

Comment:
I see the field numbers here match the normal aggchain proof. Out of curiosity, any particular reason for that?